### ISSUE TITLE
Error at compiling example from the Manual tikz-network

### ISSUE THREAD

**Issue by DiegoBneiNoah** [hackl/tikz-network#12]
As per example on page 26 of the manual (https://github.com/hackl/tikz-network/) when doing latexmk -pdflua -pvc or latexmk -pdfxe -pvc I get the following error below. Note: already tried https://github.com/hackl/tikz-network/issues/5:

```
! Package xkeyval Error: `R' undefined in families `vertex'.

See the xkeyval package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.6 \Vertices[RGB]{data/vertices_RGB.csv}

?

```
MWE:
```

\documentclass{article}
\usepackage{tikz-network}
\thispagestyle{empty}
\begin{document}
\begin{tikzpicture}
\Vertices[RGB]{data/vertices_RGB.csv}
\end{tikzpicture}
\end{document}
```

**Comment by hackl** [comment]
Unfortunately, I was not able to reproduce this error. In my case (Ubuntu 18.4, latexmk version 4.65) both options (`latexmk -pdflua -pvc` and `latexmk -pdfxe -pvc` as well as `latexmk -lualatex -pvc` and `latexmk -xelatex -pvc`) worked.

Please, can you provide me with further details? (e.g. mail me the log file)

**Note:** I used the follwoing files:

`test.tex`
```latex
\documentclass{article}
\usepackage{tikz-network}
\thispagestyle{empty}
\begin{document}
\begin{tikzpicture}
\Vertices[RGB]{test_vertices_RGB.csv}
\end{tikzpicture}
\end{document}
```
and `test_vertices_RGB.csv`
```csv
id, x, y ,size, color,opacity,label, R , G , B
 A, 0,  0, .4 , green,  .9   ,  a  ,255,  0,  0
 B, 1, .7, .6 ,      ,  .5   ,  b  ,  0,255,  0
 C, 2,  1, .8 ,orange,  .3   ,  c  ,  0,  0,255
 D, 2,  0, .5 ,   red,  .7   ,  d  , 10,120,255
 E,.2,1.5, .5 ,  gray,       ,  e  , 76, 55,255
```

**Comment by TorbjornT** [comment]
I can reproduce it with an up to date TeX Live 2019, using `pdflatex`, `xelatex` and `lualatex`. Log file from `pdflatex`, aborted run after first error: https://gist.github.com/TorbjornT/8e1df89f072e47a186a5a78965815e47

**Comment by TorbjornT** [comment]
On the other hand it worked in an old Overleaf project I had lying around where TeX Live 2017 was used, I'm guessing you don't have a fully updated system either. Log file from TL17 on Overleaf: https://gist.github.com/TorbjornT/dbd29fa3dc4ecc256773ed0a88a0572e

**Comment by u-fischer** [comment]
The problem are the spaces in the header. Due to a change in etoolbox (it uses \ifstrempty instead of \ifblank internally in a command), datatool is more picky here now. If you change the header to 

~~~~~
id,x,y,size,color,opacity,label,R,G,B
~~~~~

it compiles again.

**Comment by DiegoBneiNoah** [comment]
As @u-fischer  pointed out here and at https://tex.stackexchange.com/a/502074/178949, everything revolves around spacing formatting for the vertices_RGB.csv header although it has adaptations as indicated by @u-fischer  in the same issue mentioned in TeXStack . If there are changes in the package to allow for .csv file header spacing, it would be good to avoid interrupting the PDF file generation work, as it is easy to forget or not realize the spacing at the time of producing the .csv file.

**Comment by hackl** [comment]
Thanks for your help and pointing this out. In fact, there are problems with the new 2019 TeX Live version. I will try to fix this issue as soon as possible.

**Comment by hackl** [comment]
The issue was fixed in commit 6bfef6e. Now CSV files can be used as usual.

**Comment by DiegoBneiNoah** [comment]
> The issue was fixed in commit [6bfef6e](https://github.com/hackl/tikz-network/commit/6bfef6ecbb241b60c5447a76fdbd29af6323e4d7). Now CSV files can be used as usual.

Greatfull. As soon as I can I will test the changes to my Arch Linux / TeX Live 2019
